kazoo/tests/test_lock.py

```diff
@@ -469,6 +469,10 @@ def test_write_lock(self):
         assert gotten is False
 
     def test_rw_lock(self):
+        pytest.skip(
+            "This test is flaky "
+            "(and actually performs some ZK behavior test...)"
+        )
```

Comment:
Is there benefit to keeping this test vs outright deleting it?

Always makes me scared to just drop a locking test, but if we move it to pending, it'll be out of sight, out of mind and likely never get fixed up... I recognize investigating the source of flakiness might be outside the scope of this PR and you already spent some time on it, but I still am tempted to say we either delete it or continue to let it flake intermittently rather than skipping... 🤷‍♂️

Reply:
Re-reading the test, I was wrong, it is actually a valid one. It is not skipped anymore.

Reply:
FWIW, the test was introduced in https://github.com/python-zk/kazoo/pull/650